src/index.js

```diff
@@ -18,10 +18,11 @@ const Spruce = {
 
         this.emit('init')
 
-        document.querySelectorAll('[x-subscribe]').forEach(el => {
-            el.setAttribute('x-init', buildInitExpression(el))
-            el.removeAttribute('x-subscribe')
-        })
+        if ('Turbolinks' in window) {
```

Comment:
We can probably get rid of this if statements, since attaching an event listener doesn't depend on the window variable being present.